var/spack/repos/builtin/packages/paraview/package.py

```diff
@@ -293,7 +293,7 @@ class Paraview(CMakePackage, CudaPackage, ROCmPackage):
     patch("vtk-xdmf2-hdf51.13.2.patch", when="@5.10:5.11.0")
 
     # Fix VTK to work with external freetype using CONFIG mode for find_package
-    patch("FindFreetype.cmake.patch", when="@5.10.1:")
```

Comment:
Do you still need to patch for older freetype?

Reply:
I'm confused: this PR updates a patch file, but then comments-out the only spot that refers to the patch, such that it is never used.

Reply:
That would be a mistake on my part. Paraview/VTK's CMake will happen to work if this is commented out just by happenstance but the explicit handling here is more correct.

Reply:
Fixed and pushed.